certfields/all-fields/main.c

```diff
@@ -113,13 +113,18 @@ int main(int argc, char** argv)
     if (pubKeyTmp == NULL)
         err_sys("wolfSSL_X509_get_pubkey failed", 0);
 
+
+    /* always initialize both key structs since both are free'd at the end */
+    ret =  wc_InitRsaKey(&pubKeyRsa, 0);
```

Comment:
I suspect you could always just memset these or set a flag. This seems like a fine solution.

Reply:
I would prefer the heap hint to be NULL, not 0.